server/pbenchinacan/deploy

```diff
@@ -38,36 +38,47 @@ PB_SERVER_IMAGE_PULL_POLICY=${PB_SERVER_IMAGE_PULL_POLICY:-"newer"}
 PB_SERVER_CONTAINER_NAME=${PB_SERVER_CONTAINER_NAME:-"pbench-server"}
 
 # Locations on the host
-PB_DASHBOARD_DIR=${PB_DASHBOARD_DIR:-${GITTOP:=$(git rev-parse --show-toplevel)}/dashboard/build}
-NGINX_FAVICON=${NGINX_FAVICON:-${GITTOP:=$(git rev-parse --show-toplevel)}/server/lib/config/color-square.ico}
+GITTOP=${GITTOP:-$(git rev-parse --show-toplevel)}
+PB_DASHBOARD_DIR=${PB_DASHBOARD_DIR:-${GITTOP}/dashboard/build}
+NGINX_FAVICON=${NGINX_FAVICON:-${GITTOP}/dashboard/src/assets/logo/color-square.16x16.ico}
 PB_DEPLOY_FILES=${PB_DEPLOY_FILES:-${HOME}/Deploy}
 SRV_PBENCH=${SRV_PBENCH:-/srv/pbench}
 
 #+
 # Deployment
 #-
 
-# Update the dashboard code, removing any existing code and copying in fresh.
+# Update the Dashboard code, removing any existing code and copying in fresh.
 # We expect that the host dashboard directory already exists and contains a
 # build of the Pbench Dashboard application; however, if it is missing, we
-# deploy the Pbench Server without a Dashboard deployment.
+# deploy the Pbench Server without a Dashboard deployment.  If we do update the
+# Dashboard, then also update the Nginx favicon for it.
 podman unshare rm -r ${SRV_PBENCH}/public_html/dashboard
 if [[ -d "${PB_DASHBOARD_DIR}" ]]; then
-    podman unshare cp -r ${PB_DASHBOARD_DIR} ${SRV_PBENCH}/public_html/dashboard
+    podman run \
+        --rm \
+        --volume ${PB_DASHBOARD_DIR}:/dashboard:Z \
+        --volume ${SRV_PBENCH}:/srv/pbench:Z \
+        --entrypoint cp \
+        ${PB_SERVER_IMAGE} \
+        -r /dashboard /srv/pbench/public_html/dashboard
+    podman run \
+        --rm \
+        --volume ${NGINX_FAVICON}:/favicon:Z \
+        --volume ${SRV_PBENCH}:/srv/pbench:Z \
+        --entrypoint cp \
+        ${PB_SERVER_IMAGE} \
+        /favicon /srv/pbench/public_html/favicon.ico
```

Comment:
Since the dashboard assets are also copied at line 64, why not just symlink the `favicon.ico` to the `color-square.16x16.ico`?

Reply:
We can do that (and, I suspect we can do it from outside the container, which would be nice), but it means that we won't be able use `NGINX_FAVICON` to override the source of the favicon file (so, I'll go ahead and remove the whole `NGINX_FAVICON` mechanism (which I'm happy to do))...are you good with that?